docs/source/dataframe-parquet.rst

```diff
@@ -66,7 +66,8 @@ Engine
 ``fastparquet``.  For historical reasons this defaults to ``fastparquet`` if it
 is installed, and falls back to ``pyarrow`` otherwise. We recommend using
 ``pyarrow`` when possible. This can be explicitly set by passing
-``engine="pyarrow"``.
+``engine="pyarrow"``. No matter the preferred engine, passing the option
+explicitly is strongly encouraged.
```

Comment:
I'm not sure if I'd encourage this, I'd hope that the default setting (once pyarrow default lands) is sufficient.

Reply:
I'm good with that as long as this documentation is updated to reflect the new default when that change goes in.

Reply:
Yeah, we'll definitely keep this doc synced as changes come in.